projects/layout/components/cell/cell.styles.less

```diff
@@ -81,6 +81,10 @@
     }
 
     &[data-size='s'] {
+        &:hover {
+            border-radius: 0.5rem;
```

Comment:
probably we can use approach as below with --t-height, --t-padding
like 
`--t-radius: ...`

also please check the proprietary radiuses, maybe they fit the layout

Reply:
fixed